cmd/influxd/server_integration_test.go

```diff
@@ -1905,6 +1905,94 @@ func Test_ServerOpenTSDBIntegration_BadData(t *testing.T) {
 	}
 }
 
+func Test_ServerOpenTSDBIntegrationHTTPSingle(t *testing.T) {
```

Comment:
Can do. The other test cases I looked at originally stuck to a single test case per-test